api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/PregnantQueries.java

```diff
@@ -37,7 +39,7 @@ public static String getPregnantWhileOnArt(
         + adultInitailEncounter
         + ","
         + adultSegEncounter
-        + ") and e.encounter_datetime between :startDate and :endDate and e.location_id=:location and pe.gender='F' "
```

Comment:
@ynurmahomed seems this change was reverted with the last TxNew PR

We need to include the filter for females here. Pls re-add it.